R/ntfy.R

```diff
@@ -69,9 +104,9 @@ ntfy_send <- function(message  = "test",
   payload <- Filter(Negate(is.null), payload)
 
   if (!is.null(image)) {
-    send_image(trim_slash(server), topic, image, payload, ...)
+    send_image(trim_slash(server), topic, image, payload, config = auth, ...)
```

Comment:
Does `send_image` need an explicit `config` argument? That function already constructs a `httr_header`... does this interfere when sending an image? I'll also test.

Reply:
It seems this "just works" via the `...` magic.

Reply:
Yeah, this surprised me when I first wrote it. In the documentation for the different HTTP verbs in {httr} it says 

```
...    Further named parameters, such as query, path, etc, passed on to modify_url(). Unnamed 
       parameters will be combined with config().
```

So I think everything extra that gets passed to POST/GET/PUT/etc. somehow gets lumped into a `config` list, including preexisting config arguments.